cms/templates/course_outline.html

```diff
@@ -50,6 +50,40 @@ <h2 class="title title-3">${_("This course was created as a re-run. Some manual
     </div>
   </div>
   %endif
+
+  %if ora1_status.get('ora1_components') or ora1_status.get('ora1_advance_modules'):
+  <div class="wrapper wrapper-alert wrapper-alert-error is-shown">
+    <div class="alert announcement">
+      <i class="feedback-symbol fa fa-bullhorn"></i>
+
+      <div class="copy">
+        <h2 class="title title-3">${_("This course uses ORA1. ORA1 has been deprecated and removed from edX.")}</h2>
+
+        <p>${_("Assignments using ORA1 will not be accessible to students. Please replace any ORA1 assignments in this course with ORA2 assignments.")}</p>
+
+
+      %if ora1_status.get('ora1_components'):
+          <div>
+              <nav class="nav-related" aria-label="${_('ORA 1 Components')}">
+                  <ul>
+                      % for component_parent_key, component_display_name in ora1_status['ora1_components']:
+                        <li class="nav-item"><a href="${component_parent_key}">${_(component_display_name)}</a></li>
+                      % endfor
+                  </ul> 
+              </nav>
+          </div>
+      %endif
+
+       <p>
+        ${_("You also need to remove peergrading and combinedopenended from the \"Advanced Module List\" in ")}
+        <a href="http://studio.tnl2303.m.sandbox.edx.org/settings/advanced/edX/DemoX/Demo_Course">Studio Advanced Settings</a>.
```

Comment:
OK, I've included some suggested text below.

(Quick question: If users remove "peergrading" and "combinedopenended" before they replace their ORA1 assignments, can they still see the existing ORA1 assignments? I was assuming they can't, so I added "After you replace your ORA1 assignments" to the third text string. If they can still see existing ORA1 assignments, we can just say "EdX strongly recommends that you remove ORA1..." without the "After you replace..." part.)

This course contains ORA1 assignments. EdX no longer supports ORA1. 

Learners cannot see ORA1 assignments. If you want to use ORA in your course, you must replace the following ORA1 assignments with ORA2 assignments. 

(list of assignments)

After you replace your ORA1 assignments, edX strongly recommends that you remove ORA1 from the course advanced settings. Go to the Advanced Settings page, locate the "Advanced Module List” setting, and then delete “peergrading” and “combinedopenended” from the list.